backend/satellite_tools/repo_plugins/uln_src.py

```diff
@@ -22,55 +22,38 @@
 # pylint: disable=E0012, C0413
 import sys
 sys.path.append('/usr/share/rhn')
-from up2date_client.rpcServer import RetryServer, ServerList
 
 from spacewalk.satellite_tools.repo_plugins.yum_src import ContentSource as yum_ContentSource
 from spacewalk.satellite_tools.syncLib import RhnSyncException
+from spacewalk.satellite_tools.ulnauth import ULNAuth
 
 ULNSRC_CONF = '/etc/rhn/spacewalk-repo-sync/uln.conf'
```

Comment:
I wonder if we can import this from ULNAuth class?
Currently we have this defined twice.

Reply:
Updated. I'll merge the PR once the tests are green.